File: standalone/dzen2-hlwm/php/01-testparams.php
#!/usr/bin/php 
<?php # using PHP7
# This is a modularized config for herbstluftwm tags in dzen2 statusbar

require_once(__DIR__.'/helper.php');

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

$panel_height = 24;
$monitor = get_monitor($argv);

$dzen2_parameters = get_dzen2_parameters($monitor, $panel_height);
echo "$dzen2_parameters\n";

File: standalone/dzen2-hlwm/perl/01-testparams.pl
#!/usr/bin/perl
# This is a modularized config for herbstluftwm tags in dzen2 statusbar

use warnings;
use strict;

use File::Basename;
use lib dirname(__FILE__);

use helper;

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

my $panel_height = 24;
my $monitor = helper::get_monitor(@ARGV);

my $dzen2_parameters = helper::get_dzen2_parameters(
    $monitor, $panel_height);

print $dzen2_parameters."\n";
